.github/workflows/release.yml

```diff
@@ -205,6 +205,23 @@ jobs:
         
         git config user.name 'kiali-bot'
 
+    - name: Update CSV information
+      run: |
+        VERSION_TO_RELEASE=${RELEASE_VERSION:1}
+        DATETIME_NOW="$(date --utc +'%FT%TZ')"
+        CSV_UPSTREAM_FILE="./manifests/kiali-upstream/${VERSION_TO_RELEASE}/manifests/kiali.v${VERSION_TO_RELEASE}.clusterserviceversion.yaml"
+        CSV_COMMUNITY_FILE="./manifests/kiali-community/${VERSION_TO_RELEASE}/manifests/kiali.v${VERSION_TO_RELEASE}.clusterserviceversion.yaml"
```

Comment:
This "Update CSV information" block will change these two CSV files. I don't see where these files are 'git add'ed and committed. They should be committed to the repo prior to the branch being tagged with the release tag. Did I miss that?

Reply:
Hi @jmazzitelli, I missed that :D.

I added the files and also committed them, I tested on my fork releasing v1.51.0, in the following PR you can see the files modified in case you want to check:

https://github.com/leandroberetta/kiali-operator/pull/14/files